java/ql/test/ext/TopJdkApis/TopJdkApis.qll

```diff
@@ -0,0 +1,90 @@
+/** Provides a class to identify Top JDK APIs. */
+
+import java
+private import semmle.code.java.dataflow.FlowSummary
+private import semmle.code.java.dataflow.internal.FlowSummaryImpl as FlowSummaryImpl
+private import semmle.code.java.dataflow.ExternalFlow
+
+class TopJdkApi extends Callable {
+  TopJdkApi() {
+    exists(string apiName |
+      apiName =
+        this.getDeclaringType().getPackage() + "." + this.getDeclaringType().getSourceDeclaration() +
+          "#" + this.getName() + paramsString(this) and
+      apiName in [
+          // top 100 JDK APIs
+          "java.lang.StringBuilder#append(String)", "java.util.List#get(int)",
```

Comment:
I think it would be good to pull this string list into its own predicate `topJdkApiName(string apiname)`. If you then also expose the mapping between apiname and callable, then the test could report 1. those apinames for which there's no callable (i.e. an import is missing in the .java file), and 2. those apinames for which there _isn't_ a manual model. That way we could have an empty expected file and more confidence that all the relevant imports are present.

Reply:
Updated in https://github.com/github/codeql/pull/11572/commits/fdcaa93200c1e422b4c3239dd34e0b1f80fd31b6.
Let me know if I misunderstood anything about what you had in mind for this.